net/mwan3/files/etc/hotplug.d/iface/15-mwan3

```diff
@@ -26,24 +26,20 @@ if [ "$ACTION" == "ifup" ]; then
 	if [ "$family" = "ipv4" ]; then
 		ubus call network.interface.${INTERFACE}_4 status &>/dev/null
 		if [ "$?" -eq "0" ]; then
-			network_get_gateway gateway ${INTERFACE}_4
```

Comment:
Why do you want to delete the gateway check on this interface? Could you pleas explain this?

Reply:
Since https://github.com/openwrt/packages/pull/5033/files#diff-ccaf6dffccf2fc134d63b5c33d5ee8e1R327 . gateway is no longer used anymore.

Reply:
sorry i dont get it ?

Reply:
In such protocol (yes, OpenVPN included), no gateway are set. Example:
`default dev tun0 proto static scope link metric 1000`
So uci may fill them '0.0.0.0', or just empty string in some cases.
This won't affect currently runnable config.

Reply:
@yangfl
Thanks for clarification i will have a look in the next days on this topic. Maybe on Friday.